shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -7,6 +7,8 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
+#include "unicode/uchar.h"
```

Comment:
@dnfield since you jumped in, how do you feel about including this library?  I've been trying to think of another way to check if it's an emoji without importing this but also reinventing the wheel.  I guess it's okay?

Reply:
This seems fine to me. This is what ICU is for and there's no getting around having ICU in a flutter app